uis/MainWindow.py

```diff
@@ -86,15 +103,22 @@ def setupUi(self, MainWindow):
         QtCore.QMetaObject.connectSlotsByName(MainWindow)
 
     def retranslateUi(self, MainWindow):
-        MainWindow.setWindowTitle(QtWidgets.QApplication.translate("MainWindow", "Predictive Geology Map", None, -1))
-        self.label_output.setText(QtWidgets.QApplication.translate("MainWindow", "Output Folder:", None, -1))
-        self.pushButton_OK.setText(QtWidgets.QApplication.translate("MainWindow", "OK", None, -1))
+        MainWindow.setWindowTitle(QtWidgets.QApplication.translate("MainWindow", "MainWindow", None, -1))
+        self.pushButton_inputFileLito.setText(QtWidgets.QApplication.translate("MainWindow", "Selecionar Arquivo", None, -1))
         self.pushButton_inputFilesFeatures.setText(QtWidgets.QApplication.translate("MainWindow", "Selecionar Arquivos", None, -1))
-        self.label_inputFeatures.setText(QtWidgets.QApplication.translate("MainWindow", "Sensoriamento (features):", None, -1))
-        self.label_inputLimit.setText(QtWidgets.QApplication.translate("MainWindow", "Limite:", None, -1))
-        self.pushButton_Cancel.setText(QtWidgets.QApplication.translate("MainWindow", "Cancel", None, -1))
+        self.pushButton_OK.setText(QtWidgets.QApplication.translate("MainWindow", "OK", None, -1))
         self.pushButton_inputFileLimit.setText(QtWidgets.QApplication.translate("MainWindow", "Selecionar Arquivo", None, -1))
         self.pushButton_outputDir.setText(QtWidgets.QApplication.translate("MainWindow", "Selecionar Pasta", None, -1))
         self.label_inputLito.setText(QtWidgets.QApplication.translate("MainWindow", "Litologia (target):", None, -1))
-        self.pushButton_inputFileLito.setText(QtWidgets.QApplication.translate("MainWindow", "Selecionar Arquivo", None, -1))
+        self.pushButton_Cancel.setText(QtWidgets.QApplication.translate("MainWindow", "Cancel", None, -1))
+        self.label_inputFeatures.setText(QtWidgets.QApplication.translate("MainWindow", "Sensoriamento (features):", None, -1))
+        self.label_inputLimit.setText(QtWidgets.QApplication.translate("MainWindow", "Limite:", None, -1))
+        self.label_output.setText(QtWidgets.QApplication.translate("MainWindow", "Output Folder:", None, -1))
+        self.groupBox_options.setTitle(QtWidgets.QApplication.translate("MainWindow", "Options", None, -1))
+        self.label_maxSamples.setText(QtWidgets.QApplication.translate("MainWindow", "Maximum number of samples per class:", None, -1))
+        self.lineEdit_maxSamples.setToolTip(QtWidgets.QApplication.translate("MainWindow", "Classses with more samples than the number indicated here will be randomly selected.", None, -1))
+        self.lineEdit_maxSamples.setText(QtWidgets.QApplication.translate("MainWindow", "150", None, -1))
```

Comment:
The maximum value must always be greater than the minimum

Reply:
Cannot be negative as well

Reply:
Added check for that.